Gopkg.toml

```diff
@@ -31,5 +31,4 @@ required = ["github.com/gopherjs/gopherjs"]
   name = "github.com/skycoin/skycoin"
 
 [prune]
-  go-tests = true
```

Comment:
I believe this defaults to true, you may want to be explicit `go-tests = false` and leave a comment why (for the wasm tests)

Reply:
Adding `go-tests = false` makes `dep ensure` say `root prune options must be omitted instead of being set to false`